.github/workflows/ci.yml

```diff
@@ -0,0 +1,50 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
```

Comment:
Have you considered adding a pip cache? you can do something like [this](https://github.com/mozilla/addons-server/blob/master/.github/actions/setup-environment/action.yml#L15) and it will cache pip dependencies across builds. Still relies on network due to running in CI but it is still quite a bit faster in my experience.

Reply:
One line of code to save some external bandwidth but up to you. Just thought I'd point it out.